myuw/models/popular_link.py

```diff
@@ -1,9 +1,24 @@
+import json
 from django.db import models
 
 
 class PopularLink(models.Model):
     affiliation = models.CharField(max_length=80, null=True)
     pce = models.NullBooleanField()
+    intl_stud = models.BooleanField(default=False)
```

Comment:
Hi Stephen, 
Let me know if you prefer to have intl_stud  as one of the affiliation